scripts/linux/fv/gitlab_test_fv.sh

```diff
@@ -1,31 +1,57 @@
 #!/bin/bash -e
-#Running every test one by one
+# Running every test one by one
 
 export REPO_HOME=$PWD
-export FV_HOME=$REPO_HOME/scripts/linux/fv
-rm -rf reports && mkdir -p reports #for reports storage
+export FV_HOME=${REPO_HOME}/scripts/linux/fv
+rm -rf reports && mkdir -p reports              # folder for reports storage
 
-source $FV_HOME/olp-common.variables #variables common for all tests
+source ${FV_HOME}/olp-common.variables          # variables common for all tests
 if [ -z $production_service_secret ] || [ -z $production_service_id ] ; then
-    echo "FATAL FAILURE: production_service_id and production_service_secret are not defined"
+    echo "FATAL FAILURE: production_service_id and production_service_secret are not defined. Have you defined env.variables on GitLab?"
 fi
 
 TEST_FAILURE=0
+EXPECTED_REPORT_COUNT=6                         # expected that we generate 6 reports
 
 # Run unit tests
-$FV_HOME/gitlab-olp-cpp-sdk-authentication-test.sh || TEST_FAILURE=1
-$FV_HOME/gitlab-olp-cpp-sdk-core-test.sh || TEST_FAILURE=1
-$FV_HOME/gitlab-olp-dataservice-read-test.sh || TEST_FAILURE=1
-$FV_HOME/gitlab-olp-dataservice-write-test.sh || TEST_FAILURE=1
+${FV_HOME}/gitlab-olp-cpp-sdk-authentication-test.sh 2> errors.txt || TEST_FAILURE=1
```

Comment:
```suggestion
${FV_HOME}/gitlab-olp-cpp-sdk-authentication-test.sh 2>> errors.txt || TEST_FAILURE=1
```
otherwise errors will be overwritten